SEED-platform/seed
# Wava Fixes Part II

#### What's this PR do?
- Moved autocomplete="off" to password field in Django form generator from form tag
- Added autocomplete="off" to password fields on the profile page and PM import modal
- Added rel="noopener noreferrer" to Getting Started Guide link
- Fixed 301 redirect on the version API from the About page
- Overhauled nginx configuration (nosniff, CSP, etc) and cleared CloudFlare cache
#### How should this be manually tested?
- Check that all pages load as expected, especially ones with security policies like iframes (swagger page) and external content (map view)
#### What are the relevant tickets?
#2200 
See previous PR #2199
